salt/grafana/init.sls

```diff
@@ -78,3 +88,20 @@ grafana-import_dashboard-{{ dash }}:
     - require:
       - file: grafana-copy_dashboard_{{ dash }}
 {% endfor %}
+
+# Remove default grafana configuration
+grafana-remove_default_config:
```

Comment:
This will cause the file to always be overwritten in the next state even when no changes were introduced in the template of defaults.